account_invoice_margin/tests/test_account_invoice_margin.py

```diff
@@ -100,57 +100,3 @@ def test_invoice_refund(self):
         new_invoice = self.env["account.move"].browse(action["res_id"])
         self.assertEqual(new_invoice.invoice_line_ids.margin, 1000.00)
         self.assertEqual(new_invoice.invoice_line_ids.margin_signed, -1000.00)
-
-    def test_invoice_down_payment(self):
-        SaleAdvancePaymentInv = self.env["sale.advance.payment.inv"]
```

Comment:
Why are you removing this test? It's valuable.

Reply:
This test needs  sale dependency

Reply:
OK, I see. I suppose we need to include it in `account_invoice_margin_sale`.